src/stacks-hierarchy/Root_ChooseTicketReceiverScreen/Root_ChooseTicketReceiverScreen.tsx

```diff
@@ -0,0 +1,215 @@
+import {PhoneSignInErrorCode} from '@atb/auth';
+import {Button} from '@atb/components/button';
+import {MessageBox} from '@atb/components/message-box';
+import {FullScreenHeader} from '@atb/components/screen-header';
+import {PhoneInputSectionItem, Section} from '@atb/components/sections';
+import {ThemeText} from '@atb/components/text';
+import {RootStackScreenProps} from '@atb/stacks-hierarchy/navigation-types';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {StaticColorByType, getStaticColor} from '@atb/theme/colors';
+import {
+  LoginTexts,
+  PurchaseOverviewTexts,
+  useTranslation,
+} from '@atb/translations';
+import {OnBehalfOfTexts} from '@atb/translations/screens/subscreens/OnBehalfOf';
+import {useFocusOnLoad} from '@atb/utils/use-focus-on-load';
+import phone from 'phone';
+import {useState} from 'react';
+import {ActivityIndicator, KeyboardAvoidingView, View} from 'react-native';
+import {ScrollView} from 'react-native-gesture-handler';
+
+type Props = RootStackScreenProps<'Root_ChooseTicketReceiverScreen'>;
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+export const Root_ChooseTicketReceiverScreen: React.FC<Props> = ({
```

Comment:
There is currently a lot of duplicated code between this and `Root_LoginPhoneInputScreen`. Perhaps it is possible to create a separate component that both screens re-use?

Reply:
refactored into PhoneInput.tsx, please check 😄